scripts/xtensa-build-all.sh

```diff
@@ -306,6 +338,8 @@ do
 	( set -x # log the main commands and their parameters
 	cmake -DTOOLCHAIN="$TOOLCHAIN" \
 		-DROOT_DIR="$ROOT" \
+		"${MEU_OPENSSL_OPTION}" \
+		"${MEU_PATH_OPTION}" \
```

Comment:
Mmmm... this may be passing empty string arguments to cmake... not an issue for cmake?

Reply:
yeah shouldnt be a problem though not pretty